### ISSUE TITLE
Boston housing project's "fitting a model" code is broken

### ISSUE THREAD

**Issue by excieve** [udacity/machine-learning#195]
Commit 890eb59937711 changed `n_iter` argument to `n_splits` in the `ShuffleSplit` class constructor. Current recommended scikit-learn version throughout the course is 0.17.x, which [does not have](http://scikit-learn.org/0.17/modules/generated/sklearn.cross_validation.ShuffleSplit.html#sklearn.cross_validation.ShuffleSplit) `n_splits` yet. Since the course materials also recommend to have always up-to-date revision of this repository it contains broken code out of the box. Moreover `ShuffleSplit` constructor in scikit-learn 0.18 has a [different signature](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.ShuffleSplit.html) altogether so this change will not work there either.

Please consider reverting the mentioned commit. Thanks!

**Comment by prosanes** [comment]
:+1:

**Comment by axelglzc** [comment]
I came here to report this, it's impressing how you've done it a month ago and it's still unfixed.

**Comment by mmilburn** [comment]
I ran into this problem.  Any chance master could be based on scikit 0.17.x and commits like this one could be moved to a branch?

**Comment by adarsh0806** [comment]
```cv_sets``` definition updated to use sklearn 0.17 conventions.
